<launch>

  <node pkg = "rospkg-ogameasure"
        name = "ND287"
        type = "ND287.py">

        <param name="~az_usbport" value="['/dev/ttyUSB0']" />
        <param name="~el_usbport" value="['/dev/ttyUSB1']" />


    <remap from = '/dev/ND287/__port__/az' to = '/dev/ma24126a/ttyUSB0/az'/>
    <remap from = '/dev/ND287/__port__/el' to = '/dev/ma24126a/ttyUSB1/el'/>



  </node>

</launch>
